USER_GUIDE.md

```diff
@@ -0,0 +1,26 @@
+# OpenSearch Benchmark Workloads User Guide
+
+OpenSearch Benchmark (OSB) comes packaged with official workloads that can be found throughout this repository. This guide provides a brief overview of the organizsation of this repository as well as how OSB uses these workloads.
+
+### Contents
+- [What do branches represent](#what-do-the-numbered-branches-represent)
+- [How does OSB select which branch to use](#how-does-osb-select-which-branch-to-use)
+- [Force OSB to use a Specific Branch](#force-osb-to-use-a-specific-branch)
+
+### What do the numbered branches represent?
+
+Don't worry, these numbers are not the same [numbers](https://lostpedia.fandom.com/wiki/The_Numbers) in the series [Lost](https://en.wikipedia.org/wiki/Lost_(2004_TV_series)). Each branch -- `main`, `6`, `7`, `1`, `2`, `3` -- is associated with a specific major version of OpenSearch or Elasticsearch and contains variations of each workload.
```

Comment:
contain variations of relevant workloads appropriate for the respective branches.  Note that versions `6` and `7` can be thought of as "older" versions of OpenSearch, based on an historical version-numbering oddity.